lib/Doctrine/Adapter/Statement/Oracle.php

```diff
@@ -592,13 +592,15 @@ function($m) use (&$bind_index) { return ":oci_b_var_" . $bind_index++; },
             $query
         );
 
-        $this->statement =  @oci_parse($this->connection, $query);
+        $statement = @oci_parse($this->connection, $query);
 
-        if ( $this->statement == false )
+        if ( $statement == false )
         {
             throw new Doctrine_Adapter_Exception($this->getOciError());
         }
 
+        $this->statement = $statement;
```

Comment:
This is all done to prevent `false` being set on `$this->statement` which throws a phpstan error.  Since it checks for `false` anyhow, decided to just use an intermediate variable to store and check the result.